ovara-backend/src/test/scala/fi/oph/ovara/backend/utils/ExcelWriterSpec.scala

```diff
@@ -4056,4 +4062,727 @@ class ExcelWriterSpec extends AnyFlatSpec {
+    assert(dataRow2.getCell(5).getNumericCellValue == 0)
+    assert(dataRow2.getCell(6).getNumericCellValue == 0)
+    assert(dataRow2.getCell(7).getNumericCellValue == 0)
+    assert(dataRow2.getCell(8).getNumericCellValue == 0)
+    assert(dataRow2.getCell(9).getNumericCellValue == 1)
+    assert(dataRow2.getCell(10).getNumericCellValue == 2)
+    assert(dataRow2.getCell(11).getNumericCellValue == 2)
+    assert(dataRow2.getCell(12).getNumericCellValue == 26)
+    assert(dataRow2.getCell(13).getNumericCellValue == 0)
+    assert(dataRow2.getCell(14).getNumericCellValue == 0)
+    assert(dataRow2.getCell(15).getNumericCellValue == 0)
+    assert(dataRow2.getCell(16).getNumericCellValue == 0)
+    assert(dataRow2.getCell(17).getNumericCellValue == 0)
+    assert(dataRow2.getCell(18) == null)
+
+    // yhteensä
+    val summaryRow = sheet.getRow(3)
+    assert(summaryRow.getCell(0).getStringCellValue == "Yhteensä SV")
+    assert(summaryRow.getCell(1).getNumericCellValue == 856)
+    assert(summaryRow.getCell(2).getNumericCellValue == 287)
+    assert(summaryRow.getCell(3).getNumericCellValue == 125)
+    assert(summaryRow.getCell(4).getNumericCellValue == 0)
+    assert(summaryRow.getCell(5).getNumericCellValue == 0)
+    assert(summaryRow.getCell(6).getNumericCellValue == 0)
+    assert(summaryRow.getCell(7).getNumericCellValue == 0)
+    assert(summaryRow.getCell(8).getNumericCellValue == 0)
+    assert(summaryRow.getCell(9).getNumericCellValue == 2)
+    assert(summaryRow.getCell(10).getNumericCellValue == 57)
+    assert(summaryRow.getCell(11).getNumericCellValue == 57)
+    assert(summaryRow.getCell(12).getNumericCellValue == 287)
+    assert(summaryRow.getCell(13).getNumericCellValue == 0)
+    assert(summaryRow.getCell(14).getNumericCellValue == 0)
+    assert(summaryRow.getCell(15).getNumericCellValue == 0)
+    assert(summaryRow.getCell(16).getNumericCellValue == 0)
+    assert(summaryRow.getCell(17).getNumericCellValue == 0)
+    assert(summaryRow.getCell(18) == null)
+
+    // yksittäiset hakijat lasketaan vaan tiettyihin sarakkeisiin
+    val hakijatSummaryRow = sheet.getRow(4)
+    assert(hakijatSummaryRow.getCell(0).getStringCellValue == "Yksittäiset hakijat SV")
+    assert(hakijatSummaryRow.getCell(1).getNumericCellValue == 820)
+    assert(hakijatSummaryRow.getCell(2).getStringCellValue == "")
+    assert(hakijatSummaryRow.getCell(3).getNumericCellValue == 126)
+    assert(hakijatSummaryRow.getCell(4).getStringCellValue == "")
+    assert(hakijatSummaryRow.getCell(5).getStringCellValue == "")
+    assert(hakijatSummaryRow.getCell(6).getStringCellValue == "")
+    assert(hakijatSummaryRow.getCell(7).getStringCellValue == "")
+    assert(hakijatSummaryRow.getCell(8).getStringCellValue == "")
+    assert(hakijatSummaryRow.getCell(9).getNumericCellValue == 2)
+  }
+
+  "writeKkHakeneetHyvaksytytVastaanottaneetRaportti" should "return excel without hakutoiveet columns if naytaHakutoiveet is false" in {
```

Comment:
Tästä testin kuvauksesta vois vaihtaa ton "writeKkHakeneetHyvaksytytVastaanottaneetRaportti" -> `it`, niin ryhmittyy yhteen yläpuolella määritellyn testin kanssa (jossa on jo toi "writeKkHakeneetHyvaksytytVastaanottaneetRaportti" alussa)